x-pack/plugins/cases/public/components/case_view/metrics/index.tsx

```diff
@@ -7,26 +7,29 @@
 
 import React from 'react';
 import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner, EuiPanel } from '@elastic/eui';
+import { useGetCaseMetrics } from '../../../containers/use_get_case_metrics';
 import { CaseViewMetricItems } from './totals';
-import { CaseViewMetricsProps } from './types';
 import { CaseStatusMetrics } from './status';
+import { useCasesFeatures } from '../../cases_context/use_cases_features';
 
-export const CaseViewMetrics: React.FC<CaseViewMetricsProps> = React.memo(
-  ({ metrics, features, isLoading }) => (
+export const CaseViewMetrics = React.memo(({ caseId }: { caseId: string }) => {
+  const { metricsFeatures } = useCasesFeatures();
+  const { data, isLoading } = useGetCaseMetrics(caseId, metricsFeatures);
+  return (
     <EuiPanel data-test-subj="case-view-metrics-panel" hasShadow={false} hasBorder={true}>
```

Comment:
Now that the call is inside the component we need to render only if `metricsFeatures.length > 0`. Otherwise, an empty panel will be show to solutions that no metrics defined. Example:

<img width="2289" alt="Screenshot 2022-05-30 at 12 13 34 PM" src="https://user-images.githubusercontent.com/7871006/170960384-b3fad843-f203-48bd-84b4-7b975985a516.png">

Reply:
Thanks for catching this. I fixed it.